src/microbe_stage/Patch.cs

```diff
@@ -24,6 +24,12 @@ public class Patch
     [JsonProperty]
     public readonly int[] Depth = new int[2] { -1, -1 };
 
+    /// <summary>
+    ///   The volume of the patch, in meters cube.
```

Comment:
```suggestion
    ///   The volume of the patch, in meters cubed.
```
Isn't "meters cube" a typo? At least I've seen more "cubed" so it sounds more correct to me.

Reply:
Actually, the official naming is "cubic metre", as it seems : https://en.wikipedia.org/wiki/Cubic_metre

I'll change that.

Reply:
At least don't change the spelling to "metre" that spelling is disgusting to me as it's farther away from how the word is actually said (as a Finnish person I'm always offended when a word is not spelled the same way it is pronounced).

Reply:
Well, the thing is, the guideline is to keep by British English, whose spelling is actually "metre". I'd go against adding exception for one's sole personal taste (especially given that we'd then have to change about half the vocabulary by this metric --English be English :) )

Reply:
https://github.com/Revolutionary-Games/Thrive/pull/3107